static/js/settings_bots.js

```diff
@@ -216,6 +216,12 @@ export function update_bot_permissions_ui() {
     }
 }
 
+function show_alert_on_copy_handler(box) {
+    box.find("span.bot-info-copy-alert").text($t({defaultMessage: "Copied!"}));
+    box.find("span.bot-info-copy-alert").css("display", "block");
+    box.find("span.bot-info-copy-alert").delay(1000).fadeOut(300);
```

Comment:
We'll probably end up talking about the 1000 delay, 1 sec is probably too much lag...

...hmm, actually I notice you've just taken the same delay as in other places, eg popover.js
```js
        row.find(".alert-msg")
            .text($t({defaultMessage: "Copied!"}))
            .css("display", "block")
            .delay(1000)
            .fadeOut(300);
```

So this is probably fine.